File: app/src/main/java/com/rssclient/controllers/Tools.java
package com.rssclient.controllers;

import android.app.AlertDialog;
import android.content.Context;
import android.os.Handler;
import android.os.Looper;

public class Tools {

    public static void showErrorMessage(Context context, String string) {
        AlertDialog.Builder builder = new AlertDialog.Builder(context);
        builder.setTitle(string);
        builder.setPositiveButton("Ok", null);
        builder.create().show();
    }
}

File: app/src/main/java/com/rssclient/controllers/Tools.kt
package com.rssclient.controllers

import android.app.AlertDialog.Builder
import android.content.Context

object Tools {
    fun showErrorMessage(context: Context?, string: String?) {
        val builder = Builder(context)
        builder.setTitle(string)
        builder.setPositiveButton("Ok", null)
        builder.create().show()
    }
}